docs/pages/ios-quickstart.md

```diff
@@ -157,23 +49,39 @@ struct FormView: View {
 	
     var body: some View {
         VStack {
-            TextField("Username textfield", text: formState.username)
-            if let usernameErrorMessage = formState.usernameErrorMessage {
-                ErrorMessageView(message: usernameErrorMessage)
+            TextField(
+                "Email textfield", 
+                text: Binding(
+                    get: { formState.uiState?.email ?? "" },
+                    set: { formState.viewModel.onEmailChange(value: $0) }
+                )
+            )
+            if let emailErrorMessage = formState.uiState?.emailError {
+                ErrorMessageView(message: emailErrorMessage)
             }
-            SecureField("Password textfield", text: formState.password)
-            if let passwordErrorMessage = formState.passwordErrorMessage {
+
+            SecureField(
+                "Password textfield", 
+                text: Binding(
+                    get: { formState.uiState?.password ?? "" },
+                    set: { formState.viewModel.onPasswordChange(value: $0) }
+                )
+            )
+            if let passwordErrorMessage = formState.uiState?.passwordError {
                 ErrorMessageView(message: passwordErrorMessage)
             }
             ...
             Button {
-                // Perform sign up
+                signUp()
             } label: {
                 Text("Sign Up")
-            }.disabled(!formState.isFormValid)
+                    .foregroundColor(.black)
+            }.disabled(formState.uiState?.isFormValid == false)
         }
     }
 }
 </code></pre>
 
-And that's one of the easiest forms we can create using **FlowForms**, we don't need to care about managing the field's validation triggering nor making any complex logic to enable or disable the continue button.
+And that's one of the easiest forms we can create using **FlowForms**, we don't need to care about managing the field's validation triggering or making any complex logic to enable or disable the signup button.
+
+For more complex use cases and detailed insights into advanced functionalities and showcases of the **FlowForms** library on iOS, you can refer to the [iOS example app](https://github.com/rootstrap/FlowForms/tree/main/ExampleApp%20iOS) in the official repository. This comprehensive example provides
```

Comment:
The ending `This comprehensive example provides` feels like it should say something after `provides` 😅 Could you check?